akka-actor/src/main/scala/akka/event/EventBusUnsubscribers.scala

```diff
@@ -72,71 +72,3 @@ private[akka] object EventStreamUnsubscriber {
   }
 
 }
-
```

Comment:
Then the remaining thing in this file is `EventStreamUnsubscriber`. Please rename the file to `EventStreamUnsubscriber.scala`